web/packages/home/src/App.js

```diff
@@ -79,6 +79,7 @@ function App() {
   const [applications, setApplications] = useState(undefined)
   const [loadingAppSettings, setLoadingAppSettings] = useState(false)
   const { loading, getPagePlugin } = useContext(UiPluginContext)
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore